package.json

```diff
@@ -34,6 +36,8 @@
   },
   "devDependencies": {
     "chai": "4.3.7",
+    "eslint": "^8.50.0",
```

Comment:
Can we remove the `^` here to be consistent with other dependencies where version is fixed down (they were added using the --save-exact flag)?

Reply:
Done!